### PR TITLE
Option to hide routes from documentation

### PR DESCRIPTION
**Pull request by andyyu2004**
Note for future, include and exclude tags also makes sense for generated clients but not implemented here.

### CODE REVIEW

File: reflectapi-demo/reflectapi.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
       "serialization": [
         "json"
       ],
-      "readonly": true
+      "readonly": true,
```
Comment (avkonst) [inline reflectapi-demo/reflectapi.json:12]: can we generalize this and introduce tags field as a vector of strings? Then we would be able to include / exclude routes / types and fields by tags by providing --include-tags and --exclude-tags arguments to codegen tools.

Comment (andyyu2004) [reply reflectapi-demo/reflectapi.json:12]: Mostly done, do you expect this to apply to rust/ts too?

Comment (avkonst) [reply reflectapi-demo/reflectapi.json:12]: possibly eventually, most likely TS, C# and PHP. I can imagine we can offer generated SDKs to customers by including only features which are tagged for that customer (or group of customers). We have got the plans to narrow down the scope of APIs available significantly

Comment (avkonst) [reply reflectapi-demo/reflectapi.json:12]: but not high priority at the moment

**Pull request merged**